locations/spiders/redtag_fashion_ae_sa.py

```diff
@@ -0,0 +1,178 @@
+from typing import Any
+
+from scrapy import Spider
+from scrapy.http import Request, Response
+
+from locations.items import Feature
+
+
+class RedtagFashionAESASpider(Spider):
+    name = "redtag_fashion_ae_sa"
+    item_attributes = {
+        "brand": "Red Tag",
+        "brand_wikidata": "Q132891092",
+    }
+
+    def start_requests(self):
+        countries = ["Saudi Arabia", "UAE"]
+        for country in countries:
+            yield Request(
+                url=f"https://redtagfashion.com/locations/locations.php?country={country}",
+                meta={"country": country},
+            )
+
+    def parse(self, response: Response, **kwargs: Any) -> Any:
+        for store in response.json():
+            item = Feature()
+
+            # Basic store info
+            item["ref"] = f"{response.meta['country']}_{store['storecode']}"
+            item["name"] = store["locname"]
+
+            # Location
+            item["lat"] = store.get("lat")
+            item["lon"] = store.get("lng")
+
+            # Contact
+            item["phone"] = store.get("phone", "").strip() or None
+            item["email"] = store.get("email")
+
+            # Address
+            item["street_address"] = store.get("address")
+            item["city"] = store.get("city")
+            item["country"] = store.get("country")
+
+            # Hours
+            hours = store.get("timming")  # Note the misspelling in the API
+            if hours:
+                try:
+                    item["opening_hours"] = self.parse_hours(hours)
+                except Exception:
+                    item["opening_hours"] = None
+
+            # Additional properties
+            item["extras"] = {
+                "shop": "clothes",
+            }
+            if store.get("homeware") == "1":
+                item["extras"]["homeware"] = True
+            if store.get("cosmetics") == "1":
+                item["extras"]["cosmetics"] = True
```

Comment:
Values should be string "yes" or "no", not booleans, and `locations.categories.apply_yes_no` makes adding yes/no values easy:
```suggestion
            apply_yes_no("homeware", item, store.get("homeware") == "1")
            apply_yes_no("cosmetics", item, store.get("cosmetics") == "1")
```
Ideally, these should also be OSM tags, or especially keys already listed in `locations.categories.Extras`.